pkg/driver/constants.go

```diff
@@ -37,7 +37,7 @@ const (
 	serialNumberKey     = "serialNumber"
 	accessProtocolKey   = "accessProtocol"
 	targetNameKey       = "targetName"
-	targetScopeKey      = "targetScope"
+	targetScopeKey      = "target_scope"
```

Comment:
CSP no longer returns target_scope in the publish response.  We should be able to determine this by reading the volume attribute named target_scope.  JSON is always reported in snake_case.  However, we should still map that to camelCase and leave this alone.  What is the actual problem?

Reply:
This is what gets returned as part of volume.Config
```volume config is map[clone_of: dedupe_enabled:false destroy_on_delete:true encrypted:false folder: limit_iops:76800 limit_mbps:-1 performance_policy:SQL Server pool:default sync_on_detach:false target_scope:volume thick:false]" file="controller_server.go:824"```

Do you think something has changed while unmarshalling the object?

Reply:
Ah.  This is because there is no VolumeConfig struct defined in types.go specific to Nimble.  If there were, we could add a line like this:
	TargetScope   string                 `json:"target_scope,omitempty"`
Which would unmarshall the JSON response to TargetScope?